examples/jsm/renderers/common/Renderer.js

```diff
@@ -332,7 +332,7 @@ class Renderer {
 		// restore render tree
 
 		nodeFrame.renderId = previousRenderId;
-		this._currentRenderContext = previousRenderState;
+		this._currentRenderContext = this._previousRenderContext;
```

Comment:
I think this can be risky if we have a large call tree, set the previous one as a private variable in not enclosed in the call, this might not create an array of states as we expect.

Since this seems to be something unique to `WebGLBackend`, I think we can define it there, we would also avoid access to the `Renderer.js` private variable.

```js
WebGLBackend {

	beginRender( renderContext ) {
	
		const renderContextData = this.get( renderContext );
		renderContextData.previousContext = this._currentContext;
		
		this._currentContext = renderContext;
	
	}
	
	finishRender( renderContext ) {
	
		const renderContextData = this.get( renderContext );
		const previousContext = renderContextData.previousContext;
		
		...
	
	}

}
```